examples/open_llama/README.md

```diff
@@ -91,6 +91,11 @@ Note: You must be logged in to HuggingFace using `huggingface-cli login` to down
 
 Requirements file: [requirements-qlora.txt](requirements-qlora.txt)
 
+### Optimizing Open Llama Model with Azure Arc
+This workflow optimizes Open Llama model on Azure ML compute, and evaluate output models on your device. Please connect your device to Azure Arc by following instruction: [Self-hosted Kubernetes cluster](https://microsoft.github.io/Olive/tutorials/azure_arc.html)
+
+This example config file is [open_llama_arc.json](open_llama_arc.json).
+

```

Comment:
Can you add a requirements file? This example requires the azureml extras to run.

Reply:
Good call. Added.